### ISSUE TITLE
Spinner not looking as other input

### ISSUE THREAD

**Issue by Heliosmaster** [jquery-ui-bootstrap/jquery-ui-bootstrap#158]
Spinner - http://addyosmani.github.com/jquery-ui-bootstrap/components.html#block-spinner

![Selection_002](https://f.cloud.github.com/assets/283733/387493/5ce672d4-a6c2-11e2-84e2-240026a110c1.png)

Other input (for example) - http://addyosmani.github.com/jquery-ui-bootstrap/components.html#autocomplete

![Selection_003](https://f.cloud.github.com/assets/283733/387499/6fc99660-a6c2-11e2-8ad2-4e7a8042a77f.png)

I guess there is a problem with shadows (I quickly tested that commenting [these lines of code](https://github.com/Heliosmaster/siliwe/blob/master/public/css/custom-theme/jquery-ui-1.10.0.custom.css#L1359-L1382) produces a better result, but far from ideal).

**Comment by Heliosmaster** [comment]
For the focus behaviour, as apparently the CSS selector are not able to ascend, I managed to get it working using this javascript:

<pre lang="js">
  $(function() {
    $('.spinner').focus(function() {
      $(this).parent().addClass('ui-spinner-focused');
    });
    $('.spinner').focusout(function() {
      $(this).parent().removeClass('ui-spinner-focused');
    });
  });
</pre>

and adding to the CSS the class `ui-spinner-focused` which is none other than the same focus of the other input (as in bootstrap).

I commented out, as said before, the "Input field styling from Bootstrap". How can this be fixed by default?
